config/settings.yml

```diff
@@ -89,6 +89,31 @@
     - Metric
     - MiqQueue
     - MiqWorker
+    :vacuum_schedule: "0 2 * * 6"
+    :vacuum_tables:
+    - Vm
```

Comment:
Can we list these alphabetically?

Or you could leave it to @bdunne to do in a followup, he loves stuff like that :trollface: